templates/code-page.html

```diff
@@ -64,15 +96,15 @@ <h2 class="mt-4">{{assignment_header}}</h2>
 
   <div class="grid grid-cols-1 lg:grid-cols-2 gap-4 mt-5">
     <div>
-      <div class="w-full h-64 flex flex-col">
+      <div class="w-full h-80 flex flex-col">
```

Comment:
We should use the version in `test` here instead of this one.

Reply:
Fixed!